app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupUnlockScreen.kt

```diff
@@ -116,7 +136,7 @@ private fun SetupUnlockScreenContent(
         Spacer(modifier = Modifier.height(height = 24.dp))
         BitwardenUnlockWithBiometricsSwitch(
             isBiometricsSupported = biometricsManager.isBiometricsSupported,
-            isChecked = state.isUnlockWithBiometricsEnabled,
+            isChecked = state.isUnlockWithBiometricsEnabled || showBiometricsPrompt,
```

Comment:
This is a little wonky but we want it to look checked while the user is still enabling it.